portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,24 +28,51 @@
 public class DataServlet extends HttpServlet {
 
   private List<String> messages;
+  private List<String> comments;
 
-  @Override
-  public void init() {
-      messages = new ArrayList<>();
-      messages.add("no pomegranates");
-      messages.add("you could stop at five or six stores or just one");
-      messages.add("stop it, get some help");
-      messages.add("beep beep I'm a sheep");
-      messages.add("why are you running!?");
-      messages.add("wait... what?");
+  /**
+  * Converts a String instance to a Json string.
+  */
+  public String convertToJson(String str) {
+    
+    String json = "{";
+
+    json += "\"com\": ";
+    json += "\"" + str + "\"";
+    json += "}";        
+
+    return json;
   }
```

Comment:
You can remove this function if you're using gson library.

Reply:
Removed completed.